harness/scripts/destroy.sh.twig

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+function destroy() {
+  run docker-compose down --rmi local --volumes --remove-orphans
+
+  if [ "{{ @('docker.experimental.multiplatform_build.enabled') }}" == "yes" ]; then
+    local builder="{{ @('app.version') }}-{{ @('app.namespace') }}"
+    run docker buildx rm "${builder}"
```

Comment:
```suggestion
    run docker buildx rm "${builder}" || true
```
to avoid the scenario that the builder doesn't already exist skipping further cleanup later in the script